Source/Csla/Serialization/Mobile/ITypeMap.cs

```diff
@@ -0,0 +1,29 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ITypeMap.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Maps a type to a serializer type.</summary>
+//-----------------------------------------------------------------------
+
+namespace Csla.Serialization.Mobile;
+
+/// <summary>
+/// Maps a type to a serializer type.
+/// </summary>
+public interface ITypeMap
+{
+  /// <summary>
+  /// Gets or sets the original type.
+  /// </summary>
+  Type OriginalType { get; }
```

Comment:
Do we really need this property? Isn't it useless?
We only need `CanSerialize` and `SerializerType` or am I wrong?
Even our `TypeMap<T,S>` implementation does not need `OriginalType`. The default ctor could just use ´typeof(T)` instead of the property.

Reply:
Removed property